UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -59,6 +61,8 @@ public class CargoManager : MonoBehaviour
 
 		private static readonly List<int> randomJunkPrices = new List<int> { 5, 10, 15 };
 
+		private static List<string> researchedArtifacts = new List<string>();
```

Comment:
In the OnRoundRestart() in this script clear this list

Reply:
Oh, i've seen you've done it in Gamemanager instead, guess thats fine but could be done here

Reply:
public void EndRound()
	{
		...
		CargoManager.ClearStatics();
		...
	}

already being cleared by gamemanager

Reply:
> Oh, i've seen you've done it in Gamemanager instead, guess thats fine but could be done here instead just to keep it all in this script

Worth doing another commit to switch it over?